cime_config/tests.py

```diff
@@ -95,6 +95,13 @@
             )
         },
 
+    "e3sm_atm_stealth" : {
```

Comment:
Based on the name, will this suite contain tests of all future EAM stealth features?

Reply:
That's indeed the idea @wlin7 and I discussed. We thought we'd get it started. If there is concern about causing conflicts with PRs of other stealth features, I can rename this set.

Reply:
No that's fine.  But someone has to remember to remove a test from stealth whenever the feature becomes default "on".